osf/management/commands/populate_custom_taxonomies.py

```diff
@@ -20,13 +20,14 @@ def validate_input(custom_provider, data):
     excludes = data.get('exclude', [])
     customs = data.get('custom', {})
     merges = data.get('merge', {})
+    assert not set(includes) & set(excludes), 'There must be no overlap between includes and excludes'
     for text in includes:
         assert Subject.objects.filter(provider=BEPRESS_PROVIDER, text=text).exists(), 'Unable to find included subject with text {}'.format(text)
     included_subjects = Subject.objects.filter(provider=BEPRESS_PROVIDER, text__in=includes).include_children()
     logger.info('Successfully validated `include`')
     for text in excludes:
-        excluded = Subject.objects.get(provider=BEPRESS_PROVIDER, text=text)  # May raise not found error
-        assert excluded.object_hierarchy[0].text in includes, 'Excluded subject with text {} was not included'.format(text)
+        assert Subject.objects.get(provider=BEPRESS_PROVIDER, text=text).exists(), 'Unable to find excluded subject with text {}'.format(text)
```

Comment:
Can't use `.exists` after `.get` -- `exists` is a function on `QuerySet`, and `get` returns an object instance